recipes/poliastro/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: poliastro
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/poliastro/poliastro.git
+  git_tag: v{{ version }}
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - numpy >=1.9
+    - scipy
+    - numba >=0.23
+    - astropy >=1.0
+    - jplephem
```

Comment:
I just realized this is another conda package I am maintaining. I guess I should add another feedstock then?